client/src/components/FactQuizContainer.jsx

```diff
@@ -11,7 +12,7 @@ const FactQuizContainer = () => {
   return (
     <Grid>
       <Grid>
-        <ButtonGroup size="medium" color="secondary" aria-label="outlined button group" style={{ padding: '10px' }}>
+        <ButtonGroup size="medium" color="primary" aria-label="outlined button group" style={{ ...buttonsCss.buttons }}>
```

Comment:
We shouldn't have inline styling like this - it seems like that was still in there from when we first started but we should use `makeStyles` from [Material-UI](https://material-ui.com/styles/basics/) - there should be some good example in other components here.

Reply:
Thanks!